src/frontend/komponenter/Behandling/Personopplysninger/Barn.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { IBarn } from '../../../typer/personopplysninger';
+import TabellHeader from './TabellHeader';
+import LiteBarn from '../../../ikoner/LiteBarn';
+import { BredTd, KolonneTitler, TabellWrapper } from './TabellWrapper';
+import { EtikettInfo, EtikettSuksess } from 'nav-frontend-etiketter';
+import { EtikettLiten } from 'nav-frontend-typografi';
+import { formatertFødselsnummer } from '../../../utils/formatter';
+import styled from 'styled-components';
+
+const SpanMedVenstreMargin = styled.span`
+    margin-left: 4rem;
+`;
+
+const Barn: React.FC<{ barn: IBarn[] }> = ({ barn }) => {
+    return (
+        <TabellWrapper>
+            <TabellHeader Ikon={LiteBarn} tittel={'Barn'} />
+            <table className="tabell">
+                <KolonneTitler
+                    titler={['Navn', 'Fødselsnummer', 'Annen forelder', 'Bor med bruker']}
+                />
+                <tbody>
+                    {barn.map((barn) => {
+                        return (
+                            <tr key={barn.personIdent}>
+                                <BredTd>{barn.navn}</BredTd>
+                                <FødselsnummerBarn fødelsnummer={barn.personIdent} />
+                                <BredTd>
+                                    {barn.annenForelder &&
+                                        formatertFødselsnummer(barn.annenForelder.personIdent)}
+                                    {', '}
+                                    {barn.annenForelder && barn.annenForelder.navn}
+                                </BredTd>
+                                <BredTd>{barn.borHosSøker ? 'Ja' : '-'}</BredTd>
+                            </tr>
+                        );
+                    })}
+                </tbody>
+            </table>
+        </TabellWrapper>
+    );
+};
+
+const FødselsnummerBarn: React.FC<{ fødelsnummer: string }> = ({ fødelsnummer }) => {
+    const alder = Math.floor(
```

Comment:
Vi kan godt returnere alder fra backend i responsen

Reply:
Til info kan man dessverre ikke bruke fnr til fødselsdag. Det er stort sett riktig, men vi kan ikke garantere det så her må dere returnere fødselsdato fra backend :)

Reply:
Du har nok et godt poeng, @henninghaakonsen

Reply:
Ja, DNR kan skape trøbbel her :) Så tenker vi kan legge til fødselsdag/alder fra backend 
For å legge til den endringen trenger man å legge til att vi henter det feltet fra pdl. Bare si ifra hvis du trenger hjelp med dette.